templates/metadata-template.yaml

```diff
@@ -1,4 +1,4 @@
 releaseSeries:
-- major: MAJOR
-  minor: MINOR
-  contract: CONTRACT
+- major: 0
+  minor: 3
+  contract: v1alpha3
```

Comment:
Ouch. Why did we change this? We are parsing it as part of our `make managerless` and `make release`, so it remains generic. Otherwise we have to maintain it here and in the `Makefile`, guaranteed to get out of data.

Reply:
mmm it was not working in the way I setup e2e in test/e2e/config/packet-dev.conf.. Not sure how to fix it yet, if it has to be fixed

Reply:
This file is our own file, nothing "official" references it, so whatever uses it should be ours. 

What consumes it?

Reply:
OK, I see it [here](https://github.com/packethost/cluster-api-provider-packet/pull/49/files#diff-5440761c1f0778ad22f5716271219760R60-R62):

```yml
  files:
   - sourcePath: "../../../templates/metadata-template.yaml"
     targetName: "metadata.yaml"
   - sourcePath: "../../../templates/cluster-template.yaml"
```

What does this do? Can it have a replacements thing like the section before it?

Reply:
No it can not, the only acceptable fields are `sourcePath` and `targetName` for `files`

Reply:
That yaml file is full of hard-coded versions; this is going to be painful to maintain.

Let's not hold this up to get those fixed, so let's do the following:

1. Merge this in
1. Open a new PR with a new doc in `docs/VERSIONS.md` that lists everywhere that versions are maintained, so at least someone knows about them
1. Open a new issue that we need to templatize all of these versions and make that new doc much simpler